CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+## Setup
+1. Fork and clone the repository: `git clone git@github.com:Microsoft/vscode-java-test.git`
+2. `cd vscode-java-test/extension`
+3. Install the dependencies: `npm install`
+4. Build the Java Test Runner bundle for Java Language Server and the runner jars: `gulp build-jars`
```

Comment:
To run gulp in cli, you'll need to `install gulp-cli -g`.
So I would suggest that we use [npm scripts](https://docs.npmjs.com/misc/scripts) as the interface. You can refer to how a blank vscode extension project.